migration/README.md

```diff
@@ -0,0 +1,255 @@
+   ```shell
+   kubectl delete validatingwebhookconfigurations k8s-workload-registrar k8s-workload-registrar-webhook
+   kubectl delete service k8s-workload-registrar -n spire
+   kubectl delete clusterrolebindings k8s-workload-registrar-role-binding spire-k8s-registrar-cluster-role-binding
+   kubectl delete clusterroles k8s-workload-registrar-role spire-k8s-registrar-cluster-role
+   kubectl delete rolebinding spire-k8s-registrar-role-binding -n spire
+   kubectl delete role spire-k8s-registrar-role -n spire
+   kubectl delete configmaps k8s-workload-registrar k8s-workload-registrar-certs -n spire
+   kubectl delete secret k8s-workload-registrar-secret
+   ```
+
+## Deploy Spire Controller Manager
+
+Next we deploy the new SPIRE Controller Manager.
+
+1. Create the `ClusterSPIFFEID` CRD, `ValidatingWebhookConfiguration`, `Service`, `Roles`, and other SPIRE Controller Manager config.
+   ```shell
+   kubectl apply -f config/spire.spiffe.io_clusterspiffeids.yaml \
+                 -f config/spire.spiffe.io_clusterfederatedtrustdomains.yaml \
+                 -f config/spire-controller-manager-webhook.yaml \
+                 -f config/leader_election_role.yaml \
+                 -f config/leader_election_role_binding.yaml \
+                 -f config/role.yaml \
+                 -f config/role_binding.yaml \
+                 -f config/spire-controller-manager-config.yaml \
+                 -f config/spire-server.yaml
+   ```
+
+1. Create the `ClusterSpiffeId` custom resource. The below example will create SPIFFE IDs with this shape: `spiffe://<trust domain>/ns/<namespace>/sa/<serviceaccount>`. Only Pods with the label `spiffe.io/spiffe-id: true` will have entries auto-created. This corresponds to the `identity_template` and `identity_template_label` configurables from CRD mode Kubernetes Workload Registrar. 
+   ```shell
+   kubectl apply -f config/clusterspiffeid.yaml
+   ```
+
+   > **Note**
+   > See [FAQs](#faqs) or instructions on how to translate [label](#how-do-i-do-label-based-workload-registration), [annotation](#how-do-i-do-annotation-based-workload-registration), and [service account](#how-do-i-do-service-account-based-workload-registration) based workload registration. Also see [ClusterSPIFFEID defintion](https://github.com/spiffe/spire-controller-manager/blob/main/docs/clusterspiffeid-crd.md) for more information on how to create the shape most suitable to your environment.
+
+## Verify Spire Controller Manager Deployment
+
+First verify the Pods came up correctly. The `spire-server-0` Pod should have two containers running in it.
```

Comment:
This will only be the case if you have edited your previous pod to remove the registrar container. Might be obvious, but also might be worth mentioning that as an explicit step.

Reply:
Actually, this should be obvious from the contents of the statefulset YAML.